models/upload_sessions.go

```diff
@@ -68,6 +68,9 @@ const (
 	TreasureInDataMapComplete
 )
 
+/*MaxTimesToCheckForAllChunks is the maximum number of times we will check for the chunks in WaitForAllChunks*/
+const MaxTimesToCheckForAllChunks = 1000
```

Comment:
We should probably set this to a really high number initially just to be paranoid and to make sure we can handle big files.

Reply:
Sure, I can set it to like 5000 or something.